slidez-fe-core/src/containers/poll-editor/PollEditor.tsx

```diff
@@ -37,10 +36,18 @@ const PollEditor: React.FC<PollEditorProps> = ({ pollId }) => {
             })
         )
     }
-
-    const toggleRemoveClick = (index: number, arrayHelpers: any) => {
-        if (index > 0) {
+    const onChangeInput = (e: { target: any }) => {
+        setState(e.target.value)
+    }
+    const toggleRemoveClick = (
+        length: number,
+        index: number,
+        arrayHelpers: any
+    ) => {
+        if (length > 1) {
             arrayHelpers.remove(index)
+        } else {
+            setState({ name: '' })
```

Comment:
Hmm, won't it clear the name of poll?

Reply:
The name of the poll is the title property, so it works correctly